frontend/app_flowy/packages/appflowy_editor/lib/src/service/shortcut_event/built_in_shortcut_events.dart

```diff
@@ -274,10 +273,10 @@ List<ShortcutEvent> builtInShortcutEvents = [
     handler: doubleTildeToStrikethrough,
   ),
   ShortcutEvent(
-    key: 'Markdown link to link',
-    command: 'shift+parenthesis right',
-    handler: markdownLinkToLinkHandler,
-  ),
+    key: 'Markdown link or image',
+    command: 'shift+digit 0',
```

Comment:
No need to change the command. 

`)` = `shift+parentthesis right`.

Reply:
Addressed in [3963bd3](https://github.com/AppFlowy-IO/AppFlowy/pull/1277/commits/3963bd3e99bbac2460c31c970ecc8acd88acc486)